releases/release-announcement-template.md

```diff
@@ -0,0 +1,61 @@
+# Announcing `bindgen` $TODO_VERSION
+
+`bindgen` automatically generates Rust FFI bindings to C and C++ libraries.
+
+* [GitHub][]
+* [crates.io][]
+
+Upgrade to this release by updating your `Cargo.toml`:
+
+```toml
+bindgen = "$TODO_VERSION"
+```
```

Comment:
I would put this above the links to GH/crates.io since it is probably more useful for many users.